kong/plugins/acl/handler.lua

```diff
@@ -69,8 +69,9 @@ function ACLHandler:access(conf)
     end
 
     if to_be_blocked == false then
-      -- we're allowed, so go and convert 'false' to the header value
-      to_be_blocked = table_concat(consumer_groups, ", ")
+      -- we're allowed, so go and convert 'false' to the header value, if we need it.
+      -- if we don't need it, set a dummy value to save memory for potentially very long strings
+      to_be_blocked = conf.hide_groups_header and "" or table_concat(consumer_groups, ", ")
```

Comment:
@jeremyjpj0916 Will you bring these lines back to being under 80 chars long? Thanks!

Reply:
```
      -- we're allowed, convert 'false' to the header value, if needed.
      -- if not needed, set dummy value to save mem for potential long strings.
      to_be_blocked = conf.hide_groups_header and "" 
                      or table_concat(consumer_groups, ", ")
```
Tried to shorten wording and newline the long line of code properly, this work or any further change desired?